client.go

```diff
@@ -768,7 +772,7 @@ func (client *Client) CreateAPIKey() (*Response, error) {
 	if err != nil {
 		return nil, err
 	}
-	var key map[string]interface{}
+	var key APIKey
```

Comment:
Do the tests still pass with this change? I have a feeling they won't.

Reply:
I changed it where it needed changing and fixed the tests